lib/engine/game/g_1870/step/price_protection.rb

```diff
@@ -60,24 +66,23 @@ def process_buy_shares(action)
               price: price
             )
 
-            @round.goto_entity!(player)
+            @round.goto_entity!(player) if @round.entities[@round.entity_index].player?
```

Comment:
Why this change? Or is it to stop the entity changing in an OR?

Reply:
If player B sells a share of a corp of player A, and player A protects the share, the game continues to the player next to player A (instead of the one next to player B!)

But this only makes sense in the SR

Reply:
Please add a comment explaining.